### PR TITLE
Minor tweaks for plotting

### PR DESCRIPTION
**Pull request by andreypz**
Just a few changes to make it work on my outputs

### CODE REVIEW

File: src/BTVNanoCommissioning/utils/plot_utils.py [hunk 1]
```diff
@@ -731,6 +731,7 @@ def autoranger(hist, flow=None):
                 axis[-1] + (axis[1] - axis[0]) * 3,
             ]
         )
+    mins,maxs=0,1
```
Comment (Ming-Yan) [inline src/BTVNanoCommissioning/utils/plot_utils.py]: I think the `maxs` should set as default axis range. namely `len(val)-1`

File: plotting/comparison.py [hunk 2]
```diff
@@ -135,7 +135,7 @@
         plotratio(
             collated[c][var],
             collated[refname][var],
-            denom_fill_opts=None,
+            denom_fill_opts={},
```
Comment (Ming-Yan) [inline plotting/comparison.py:138]: If we want to change the default with `denom_fill_opts={}`, the [setup](https://github.com/cms-rwth/CoffeaRunner/blob/1ed0d8f08696155bba908984af78051085c8ffca/src/BTVNanoCommissioning/utils/plot_utils.py#L589) in `plot_util.py`  should change to `if denom_fill_opts is not {}:`

**Review by Ming-Yan — CHANGES_REQUESTED** [review]
Hi @andreypz ,  thanks for the proposed changes, there are some minor changes need to be done then I can merge this PR

**Review by Ming-Yan — CHANGES_REQUESTED** [review]
Hi @andreypz ,  thanks for the proposed changes, there are some minor changes need to be done then I can merge this PR

**Review by Ming-Yan — CHANGES_REQUESTED** [review]
Hi @andreypz ,  thanks for the proposed changes, there are some minor changes need to be done then I can merge this PR

File: src/BTVNanoCommissioning/utils/plot_utils.py (hunk 1 above)
Comment (andreypz) [reply src/BTVNanoCommissioning/utils/plot_utils.py]: Ok, done

File: plotting/comparison.py (hunk 2 above)
Comment (andreypz) [reply plotting/comparison.py:138]: Ok, changed the plot_utils.py

**Pull request merged**